.nix-helpers/overlays.nix

```diff
@@ -34,6 +34,14 @@ let
                     ! any (flip hasPrefix (baseNameOf path)) [ "dist" ".ghc" ];
                 };
 
+              vteWithSixel = self.vte.overrideAttrs ( vself: vsuper: {
+                mesonFlags = vsuper.mesonFlags ++ [ "-Dsixel=true" ];
+                src = super.pkgs.fetchurl {
+                  url = "https://github.com/GNOME/vte/archive/8ef3f6b2f8043d28cbc82520eb094f09333b26ae.tar.gz";
```

Comment:
@junjihashimoto What version of VTE is this?  Is this just `master` as of today?